backend/app/routers/files.py

```diff
@@ -207,6 +251,7 @@ async def update_file(
             }
         }
         update_record(es, "file", doc, updated_file.id)
+        await _resubmit_file_extractors(file_id, credentials, db, rabbitmq_client)
```

Comment:
It might be worthy to add a return message to the `_resubmit...` function so you can get some status back? If resubmit failed there will be a way to return that info to the client?

Reply:
I made the method return a list. If any extractor succeeds, it goes into a succeeds list and the ones that fail go into a fail list.